packages/gatsby-theme-patternfly-org/components/sideNav/sideNav.css

```diff
@@ -27,3 +27,11 @@
 }
 
 .ws-page-sidebar .pf-c-nav { --pf-c-nav__link--before--BorderBottomWidth: 0; }
+.ws-page-sidebar .pf-c-nav { --pf-c-nav--m-light__link--hover--BackgroundColor:
```

Comment:
Can't you just write all of these rules inside one selector like this?
```
/* Make sidenav selected background consistent with PageSection background */
.ws-page-sidebar .pf-c-nav {
  --pf-c-nav--m-light__link--hover--BackgroundColor: var(--pf-global--BackgroundColor--light-200);
  --pf-c-nav--m-light__link--focus--BackgroundColor: var(--pf-global--BackgroundColor--light-200);
  --pf-c-nav--m-light__link--m-current--BackgroundColor: var(--pf-global--BackgroundColor--light-200);
  --pf-c-nav--m-light__link--active--BackgroundColor: var(--pf-global--BackgroundColor--light-200);
}
```

Reply:
Yes, you are right, I had tested them one at at time and  should have thought of moving them to one selector when I was done.